protocols/kad/src/behaviour.rs

```diff
@@ -2497,3 +2553,8 @@ pub enum RoutingUpdate {
     /// peer ID).
     Failed,
 }
+
+/// The maximum number of local addresses as reported by
+/// other peers that the behaviour tracks. The behaviour
+/// always tracks all its listen addresses.
```

Comment:
```suggestion
/// The maximum number of local external addresses. When reached any further externally reported addresses are ignored. The behaviour always tracks all its listen addresses.
```

Nit pick: Wouldn't this be _more_ correct?